apis/apis.json

```diff
@@ -54,9 +59,13 @@
     "type": "grpc",
     "description": "Recommended Google client library to access the Google Cloud Datastore API, which accesses the schemaless NoSQL database to provide fully managed, robust, scalable storage for your application.",
     "tags": [ "Datastore" ],
+    "dependencies": {
+      "Grpc.Core": "1.4.0",
+      "Google.Api.Gax.Grpc": "2.0.0"
+    },
     "testDependencies": {
-      "Google.Cloud.ClientTesting": "",
-      "Google.Api.Gax.Grpc.Testing": ""
+      "Google.Cloud.ClientTesting": "project",
+      "Google.Api.Gax.Grpc.Testing": "2.0.0"
```

Comment:
To make sure we test against the version of GAX that the package depends on.

Suppose we have a bug in GAX 2.0.0 that is inadvertently fixed in 2.1.0. If we depend on Google.Api.Gax.Grpc.Testing 2.1.0, that will force GAX 2.1.0 as a dependency, so we'll never see the bug, despite it still being present in the released combination.